projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/datepicker-docs/datepicker-docs.component.html

```diff
@@ -509,4 +509,44 @@ <h4 class="go-heading-4 go-heading--underlined">Code</h4>
     </div>
     
   </go-card>
+
+  <go-card id="form-datepicker-min-date" class="go-column go-column--100">
+    <ng-container go-card-header>
+      <h2 class="go-heading-2 go-heading--no-wrap">Component Reset Date</h2>
+      <go-copy cardId="form-datepicker-min-date" goCopyCardLink></go-copy>
+    </ng-container>
+    <div class="go-container" go-card-content>
+
+      <div class="go-column go-column--100">
+        <p class="go-body-copy go-body-copy--no-margin">
+          The datepicker components placeholder attribute can be set via
+          <code class="code-block--inline">this.form.reset()</code>
+          and the value becomes null.
+          <code class="code-block--inline">mm/dd/yyyy</code>
+        </p>
+      </div>
+
+      <div class="go-column go-column--100 go-column--no-padding">
+        <div class="go-container">
+          <div class="go-column go-column--50">
+            <h4 class="go-heading-4 go-heading--underlined">View</h4>
+           <go-datepicker
+            [control]="dobReset"
+            label="Date of Birth">
+          </go-datepicker>
+            <br/>
```

Comment:
Looks like the indentation is off a bit here.

Reply:
I have indented the code. Please let me know if anything need to update from my end